hbase-common/src/main/java/org/apache/hadoop/hbase/ChoreService.java

```diff
@@ -283,7 +279,14 @@ private synchronized boolean requestCorePoolIncrease() {
     // amongst occurrences of the same chore).
     if (scheduler.getCorePoolSize() < scheduledChores.size()) {
       scheduler.setCorePoolSize(scheduler.getCorePoolSize() + 1);
-      printChoreServiceDetails("requestCorePoolIncrease");
+
+      if (LOG.isTraceEnabled()) {
+        LOG.trace("requestCorePoolIncrease");
```

Comment:
Personal taste.  It seems overkill to have a method just for logging.  Nicer when I can just see the logging and move on with my life, not having to jump into a method.  If that logging method was used 10 times, I'd probably keep it, but for 2x, it doesn't feel worth it.

Reply:
... it also gets confusing because folks need to know who is responsible for the logging guards... internal or external to the method.  I've seen it done both ways and I've seen where people then don't include guards when they should or they double-bag it.